src/ext-time.adoc

```diff
@@ -11,13 +11,29 @@ struct sbiret sbi_set_timer(uint64_t stime_value)
 ----
 
 Programs the clock for next event after *stime_value* time. *stime_value*
-is in absolute time. This function must clear the pending timer interrupt
-bit as well.
+is in absolute time.
 
 If the supervisor wishes to clear the timer interrupt without scheduling
-the next timer event, it can either request a timer interrupt infinitely
-far into the future (i.e., (uint64_t)-1), or it can instead mask the timer
-interrupt by clearing `sie.STIE` CSR bit.
+the next timer event, it can request a timer interrupt infinitely far
+into the future (i.e., (uint64_t)-1). Alternatively, to not receive timer
+interrupts, they can be masked by clearing `sie.STIE` CSR bit.
```

Comment:
Or it may mask the timer by clearing the `sie.STIE` CSR bit.

Reply:
Done